pod/annotations.go

```diff
@@ -206,6 +206,15 @@ const (
 	AnnotationKeyPredRuntimeQuantiles    = "runtime.predictions.titus.netflix.com/quantiles"
 	AnnotationKeyPredRuntimeModelVersion = "runtime.predictions.titus.netflix.com/model-version"
 	AnnotationKeyPredRuntimeModelID      = "runtime.predictions.titus.netflix.com/model-id"
+
+	// mockVK
+	AnnotationKeyTitusParameterMockVkPrepareTime = "titusParameter.mockVK.prepareTime"
+	AnnotationKeyTitusParameterMockVkRunTime     = "titusParameter.mockVK.runTime"
+	AnnotationKeyTitusParameterMockVkKillTime    = "titusParameter.mockVK.killTime"
+
+	AnnotationKeyPodParameterMockVkPrepareTime = "github.com.netflix.titus.executor/killTime"
```

Comment:
this annotation key looks pretty strange. github.com??

How about `pod.netflix.com/mockPodKillTime` etc

Reply:
And use `mockPod` everywhere, no more `mockVK`

Reply:
Or maybe `mockPod.netflix.com/killTime`?